canvas_sdk/value_set/custom.py

```diff
@@ -22,10 +22,47 @@ class DysrhythmiaClassConditionSuspect(ValueSet):
     EXPANSION_VERSION = "CanvasHCC Update 2024-11-04"
 
     ICD10CM = {
-        "I420", "I421", "I422", "I423", "I424", "I425", "I426", "I427", "I428", "I429", "I470",
-        "I471", "I4710", "I4711", "I4719", "I472", "I4720", "I4721", "I4729", "I479", "I480",
-        "I481", "I4811", "I4819", "I482", "I4820", "I4821", "I483", "I484", "I4891", "I4892",
-        "I4901", "I4902", "I491", "I492", "I493", "I4940", "I4949", "I495", "I498", "I499"
+        "I420",
```

Comment:
I was advised against letting `black` format the lists of codes for readability reasons. No problem with this as is, but whichever way we go, at some point we should reformat everything for consistency.